dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/config_bar_chart_styles.tsx

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useMemo, useCallback } from 'react';
+import { EuiAccordion, EuiSpacer } from '@elastic/eui';
+import { ButtonGroupItem } from './config_button_group';
+import { IConfigPanelOptionSection } from '../../../../../../../../common/types/explorer';
+
+export const ConfigBarChartStyles = ({
+    visualizations,
+    schemas,
+    vizState,
+    handleConfigChange,
+    sectionName,
+    sectionId = 'chartStyles'
+}: any) => {
+    const { data } = visualizations;
+    const { data: vizData = {}, metadata: { fields = [] } = {} } = data?.rawVizData;
+
+    const handleConfigurationChange = useCallback(
+        (stateFieldName) => {
+            return (changes) => {
+                handleConfigChange({
+                    ...vizState,
+                    [stateFieldName]: changes,
+                });
+            };
+        },
+        [handleConfigChange, vizState]
+    );
+
+    /* To update the schema options based on current style mode selection */
+    const currentSchemas = useMemo(() => {
+        if (vizState?.orientation === 'h') {
```

Comment:
what does the 'h' represent here? Horizontal ?

Reply:
Hi @kavithacm , here 'h' and 'v' is for horizontal and vertical respectively.